.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Description
+
+Please explain the changes you made, including a reference to the related issue if applicable
+
+### Checklist
+- [ ] Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
```

Comment:
I feel this does not give away that testing will fail if the patch is not formatted.
All the other criteria are soft, but this one will be enforced (from now on).